### DISCUSSION TITLE
Question: Are you affiliated with "Cocaine Bear" by any chance?

### DISCUSSION THREAD

**Discussion by creatorrr** [beartype/beartype#392] [Q&A]
Jk. Thanks for the amazing library. It has saved my bacon a bunch of times! 🐻 

*****

![[Cocaine Bear](https://www.imdb.com/title/tt14209916/)](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWo2Y2Z4NjA2dTVmemdxYzkyYzM0ZXN3MG1tOGhteXFkMjlpZDdveCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/Izy9JPexCeOERHJ3As/giphy.gif)

**Comment by leycec** [comment]
![](https://media.tenor.com/4hc1CkaoQRkAAAAM/wildlife-bear.gif)
<sup>Dr. Biggles, the @beartype mascot, rolls around on suspicious drums filled with "confectionery sugar"</sup>

**Srsly, though.** I am so frightened by this real-world footage that I had no choice but to convert your hard-hitting question #390 into a "discussion." Let's pretend this never happened. No more confectionery sugar, Dr. Biggles! :rofl:
